R/dot-net-wrapper.R

```diff
@@ -6,113 +6,47 @@
 #' sim <- loadSimulation(system.file("extdata", "Aciclovir.pkml", package = "ospsuite"))
 #'
 #' # looking at a reference to `.NET` simulation object
-#' sim$ref
+#' sim$pointer
 #'
 #' # create a new instance of `DotNetWrapper` class using this reference
-#' DotNetWrapper$new(sim$ref)
+#' DotNetWrapper$new(sim)
 #'
 #' @export
 DotNetWrapper <- R6::R6Class(
   "DotNetWrapper",
-  inherit = Printable,
+  inherit = NetObject,
```

Comment:
For now, `NetObject` does not inherit from `Printable` but re-implements its functionality. Has some drawbacks, but I did not want make the utils package a dependency yet.